### ISSUE TITLE
Any angle greater than zero for rotation on the X and Y axis raises an error while the images are generated

### ISSUE THREAD

**Issue by LFBJC** [faustomorales/keras-ocr#186]
Anytime I try to generate training data with any interval the parameters rotationX or rotationY of the function keras_ocr.data_generation.get_image_generator() it raises an error in the compute_transformed_contour function, when I debugged I found out that this has to do with a "slots" array inside the function that gets empty after this step: `slots_filtered = slots_pretransform[(areas > minarea * spacing * spacing) & inside]` because `slots_filtered = slots_pretransform[inside]` is empty. The image generator only works if I set rotationX and rotationY to 0
